test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
+    (let [log-messages (atom [])]
+      (tk-log-testutils/with-logging-to-atom tk-log/root-logger-name log-messages
+        (let [app (tk-bootstrap/bootstrap-services-with-config
+                   [jetty9-service]
+                   {:webserver {:port 8080}})]
+          (tk-app/stop app)
+          (tk-app/stop app)))
+      ;; if we try to unregister the mbeans more than once Jetty will log a bunch
+      ;; of very loud and terrifying InstanceNotFoundExceptions
```

Comment:
but this is no longer happening with this fix, right? so these exceptions should no longer be logged? it might be good to change the phrasing of this, because right now this comment reads to me like we still expect this to happen.